setup.sh

```diff
@@ -34,8 +35,13 @@ else
             fi
 
             conda install -y -c conda-forge mpi mpi4py petsc petsc4py
+            "$CONDA_PREFIX/bin/pip" install tox
             # If complex number support is needed
             #conda install -y -c conda-forge mpi mpi4py "petsc=*=*complex*" "petsc4py=*=*complex*"
+
+            #  Environment variables
+            export PYTHONPATH="$CONDA_PREFIX/lib/python3.10/site-packages"
+            unset PETSC_DIR
```

Comment:
The user is a grown-up. There must be a good reason If `PETSC_DIR` is in the environment provided by the user. So I would not remove it.
```suggestion
```